test/plugin/scenarios/gateway-scenario/config/expectedData.yaml

```diff
@@ -52,30 +52,26 @@ segmentItems:
     segments:
       - segmentId: not null
         spans:
-          - operationName: GATEWAY/provider_route
+          - operationName: HEAD:/provider/b/healthCheck
             operationId: 0
             parentSpanId: -1
             spanId: 0
             spanLayer: Http
             startTime: nq 0
             endTime: nq 0
-            componentId: 61
+            componentId: 67
             componentName: ''
             isError: false
-            spanType: Exit
-            peer: not null
+            spanType: Entry
```

Comment:
Where is the exit span?

Reply:
There is something wrong with this place, I will check again